src/twitch.js

```diff
@@ -1,12 +1,13 @@
 const request = require('./bin/request.js');
 const url = require('url');
 const Promise = require('bluebird');
+const qs = require('querystring');
```

Comment:
oh my bad, the `query-string` npm module came up first when I searched, and the method name (`.stringify()`) is the same.

sorry!